depends/packages/packages.mk

```diff
@@ -5,6 +5,7 @@ qt_packages = qrencode protobuf zlib
 
 qt_x86_64_linux_packages:=qt expat dbus libxcb xcb_proto libXau xproto freetype fontconfig libX11 xextproto libXext xtrans
 qt_i686_linux_packages:=$(qt_x86_64_linux_packages)
+qt_arm_linux_packages:=$(qt_x86_64_linux_packages)
```

Comment:
Since there's no point in making the distinction here, let's just get rid of it.

```qt_x86_64_linux_packages``` can just become ```qt_linux_packages```, then ``` qt_i686_linux_packages``` and ```qt_arm_linux_package``` are no longer needed.

Reply:
The patch does not work yet for aarch64, so I think we should keep it like this until that it is fixed.

Reply:
Making the suggested change above will make it so that no change is needed here for aarch64.

Reply:
Substituting `qt_x86_64_linux_packages` with `qt_linux_packages` means that qt will be built for all Linux architectures. When I do that and remove the two lines below, a `aarch64-linux-gnu` depends build will also try to build qt, but will fail as I mentioned in #13495 .

Reply:
Yes, ok, I forgot we purposefully limited that way.